src/Microsoft.ML.Recommender/MatrixFactorizationTrainer.cs

```diff
@@ -89,8 +89,17 @@ namespace Microsoft.ML.Trainers
     public sealed class MatrixFactorizationTrainer : TrainerBase<MatrixFactorizationPredictor>,
         IEstimator<MatrixFactorizationPredictionTransformer>
     {
+        public enum LibMFLossFunctionType { SquareLossRegression=0, SquareLossOneClass=12 };
+
         public sealed class Arguments
         {
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Loss function minimized for finding factor matrices. " +
+                "Two values are allowed, 0 or 12. The values \"0\" means traditional collaborative filtering problem with squared loss. " +
+                "The value \"12\" triggers one-class matrix factorization for implicit-feedback recommendation problem.")]
```

Comment:
please put this text in the XML comments of this argument. I know we have zip xml comments in args classes, but at least let's start doing so on the ones we add.

Reply:
I am not following. Could you elaborate?